src/main/java/com/blackduck/integration/sca/upload/client/uploaders/UploaderFactory.java

```diff
@@ -50,11 +58,24 @@ private ArtifactsUploader createArtifactsUploader(String urlPrefix) {
         return new ArtifactsUploader(uploaderConfig.getUploadChunkSize(), createFileUploader(urlPrefix), createUploadValidator());
     }
 
+    /**
+     * Construct the uploader for BDBA uploads.
+     *
+     * @param urlPrefix
+     *            Used to create {@link UploadRequestPaths}.
+     * @return the {@link BdbaUploader} created.
+     */
+    public BdbaUploader createBdbaUploader(String urlPrefix) {
```

Comment:
Doesn't appear that this uploader has a test written for it.  Can a test class be written for this uploader?

Reply:
Added test for BdbaUploader.